deployer/scripts/uuid_migrate.sh

```diff
@@ -88,7 +91,7 @@ function run_uuid_migration() {
     [[ "${OPS_PROJECTS[@]}" =~ "$index" ]] && continue
 
     uid=$(oc get project "$index" -o jsonpath='{.metadata.uid}')
-    create_alias $index $uid
+    create_alias ${PROJ_PREFIX}$index $uid
```

Comment:
I'm not sure if this is the result of the github layout but it looks like these changes are still in for the uuid_migrate.sh.
`create_alias` takes in the old format project name and its uuid, if we are sending it "project.$projectname" $uid it won't be able to find that project to create an alias for.

Unless we are planning on reusing the uuid_migrate script to migrate again to a new format, I dont think the changes to this file should be kept.

https://github.com/nhosoi/origin-aggregated-logging/blob/cb32d37d2f10d2ccd3f8cdacbacbabf4de924957/deployer/scripts/uuid_migrate.sh#L23